udffsck/Makefile.am

```diff
@@ -1,4 +1,36 @@
-noinst_PROGRAMS = udffsck
-udffsck_SOURCES = main.c
+if WORDS_LITTLEENDIAN
+sbin_PROGRAMS = udffsck
+udffsck_LDADD = $(top_builddir)/libudffs/libudffs.la
+udffsck_SOURCES = main.c utils.c utils.h udffsck.c udffsck.h options.c options.h log.c log.h ../include/ecma_167.h ../include/osta_udf.h ../mkudffs/mkudffs.h ../mkudffs/defaults.h ../mkudffs/file.h ../libudffs/crc.c ../include/libudffs.h ../include/udf_endian.h
```

Comment:
Looks suspicious. You need header files from mkudffs (which declare mkudffs functions), but not their implementation?

Reply:
You are right. In some early development stages I wanted to use something from there, but it was removed later.

Removed here:  f1aa7e7